packages/channels/botpress-channel-slack/src/actions.js

```diff
@@ -60,6 +60,13 @@ const createAttachments = (channelId, attachments, options = {}) => {
   validateChannelId(channelId)
   validateAttachments(attachments)
 
+  attachments = attachments.map(attachment => {
+    const defaultPrefix = attachment.text ? attachment.text.replace(/\s/g, '_') : 'slack_attachment'
+    const defaultCallbackId = `${defaultPrefix}:${new Date().getTime()}`
```

Comment:
`new Date().getTime()` => `Date.now()`

They produce the same, but `Date.now()` is the modern way to do it.

Reply:
@sindresorhus ty)